### ISSUE TITLE
Game menu: why we need "Back" button ?

### ISSUE THREAD

**Issue by reyandme** [Kromster80/kam_remake#244]
Currently, on F1 menu we have "Back" button:
![image](https://cloud.githubusercontent.com/assets/5173273/22846169/47dae0dc-eff7-11e6-825d-d47c27b168ac.png)

1. When F1 menu is opened there is always some "tool" selected (road/corn/wine/erase/house). 
2. On right mouse button - we close this menu, and can't see tools anymore. 

Because of 1. point, we can not see opened F1 menu all the time, because our cursor become cyan quad/cyan house plan, and we could occasionaly place some road/house plan, which is unwanted. 
That means to build smth you either have to build it in a row, one after another, or you have to press F1 button all the time. 

Other menus can be opened and you can watch them for long time without UI problems (cyan quads etc.)

Why can't we have always opened building menu, like any other menu? Currenty we have this "Back" button, which only show us another menu buttons (and closes F1, as well as any other, menu page, releasing cursor from being "cyanquaded"). 
Instead of this button we can place all menu buttons there. Then to cancel build tool - you either have to click on it or just click right mouse btn. After some time you can choose any tool again and continue city planning. 

Btw in Map Editor we have exactly the same menu as I described. And its very convinient.

The only reason why Back button is still here is because it was there in the original game. Can we somehow alter this (optionally?) ? 

Or may be can not close F1 menu on right mouse button click, only unselect the tool first, an only second right click will reset menu to all 4 buttons. 

Last option is to add new hotkey for this, then it could be possible to unselect tool with, let say, 4th mouse btn (I added support of 4th and 5th mouse button in #139 )

**Comment by Kromster80** [comment]
When we open F1 menu, chances are we are going to build something - a house or a road.
With F2-F3 menus, we will look at them and think, possibly not changing anything.
F4 menu has no gameplay effects.

This is why Road is selected by default in F1 menu.
We don't spend time looking at F1 menu and think.

TL;DR: Leave it as is.

**Comment by Kromster80** [comment]
"Back" button is there because it clearly indicates in which menu are we. Without it we would have no clear way to return back.

I'm against MapEd-styled menu in the game, because we dont have clear indication of which menu we are in.

**Comment by reyandme** [comment]
Ok, after skype conversation I will try to improve MapEd menu first, then we will see
